src/main/java/com/gregtechceu/gtceu/syncsystem/ManagedSyncBlockEntity.java

```diff
@@ -43,13 +44,13 @@ protected final void saveAdditional(CompoundTag tag) {
     }
 
     @Override
-    public void load(CompoundTag tag) {
+    public final void load(CompoundTag tag) {
         super.load(tag);
-        getSyncDataHolder().deserializeNBT(tag, false);
+        if (tag.contains("machine")) tag.merge(tag.getCompound("machine"));
```

Comment:
Is this the right place to be doing this? Putting it in `ManagedSyncBlockEntity` feels like an overreach for performing this migration since other things can make use of this class without being machines.